dataflowengine/src/main/scala/io/shiftleft/dataflowengine/passes/reachingdef/ReachingDefPass.scala

```diff
@@ -5,31 +5,30 @@ import java.nio.file.Paths
 import gremlin.scala._
 import io.shiftleft.Implicits.JavaIteratorDeco
 import io.shiftleft.codepropertygraph.Cpg
-import io.shiftleft.codepropertygraph.generated.nodes.FieldIdentifier
+import io.shiftleft.codepropertygraph.generated.nodes.{Call, FieldIdentifier, Identifier, Literal, StoredNode}
 import io.shiftleft.codepropertygraph.generated.{nodes, _}
 import io.shiftleft.passes.{CpgPass, DiffGraph, ParallelIteratorExecutor}
 import io.shiftleft.semanticcpg.utils.{ExpandTo, MemberAccess}
 import io.shiftleft.semanticcpg.language._
 
+import scala.collection.mutable
 import scala.jdk.CollectionConverters._
 
 class ReachingDefPass(cpg: Cpg) extends CpgPass(cpg) {
-  var dfHelper: DataFlowFrameworkHelper = _
+  lazy val dfHelper: DataFlowFrameworkHelper = new DataFlowFrameworkHelper(cpg.graph)
```

Comment:
Please do not make this lazy. 
1. The Helper class is super light weight.
2. There is no reason to believe that a ReachingDefPass is often time created but run() is not called.

Reply:
fully agree, not sure why i even did that. change pushed, please re-review.